cmd/idmd/db/dump/dump.go

```diff
@@ -0,0 +1,65 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright 2021 The LibreGraph Authors.
+ */
+
+package dump
+
+import (
+	"bytes"
+	"encoding/binary"
+	"encoding/gob"
+	"fmt"
+	"os"
+
+	"github.com/go-ldap/ldap/v3"
+
+	"github.com/spf13/cobra"
```

Comment:
Strange import grouping. Should be `std, external, local`.

Reply:
Hm, actually this file isn't supposed to be part of the PR at all. It's a leftover from an earlier experiment :man_facepalming: I'll clean that up.